### ISSUE TITLE
Region's pad function should reduce the min

### ISSUE THREAD

**Issue by medakk** [SCIInstitute/ShapeWorks#523]
In the executable branch, [Image.h](https://github.com/SCIInstitute/ShapeWorks/blob/e2a8e733a1d8212a92b64487df1fd3a8b38f2964/Libs/Image/Image.h):

```c++
    /// grows or shrinks the region by the specified amount
    void pad(int padding) {
      for (auto i=0; i<3; i++) {
        min[i] += padding;
        max[i] += padding;
      }
    }
```

I'm not familiar with this but that should probably be `min[i] -= padding`.

**Comment by archanasri** [comment]
It should. Thank you!

**Comment by archanasri** [comment]
Fixed with commit 3a1383a
